### PR TITLE
Add ownership transfer for kvstore commands

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

File: attributes/network/cddl/7_kvstore_commands.cddl [hunk 1]
```diff
@@ -1,26 +1,36 @@
 ; tag::types[]
 event-type-put = [7, 0]
 event-type-disable = [7, 1]
+event-type-disable = [7, 2]
```
Comment (fmorency) [inline attributes/network/cddl/7_kvstore_commands.cddl]:
```suggestion
event-type-transfer = [7, 2]
```

File: attributes/network/cddl/7_kvstore_commands.cddl [hunk 2]
```diff
@@ -55,3 +65,18 @@ kvstore.disable@param = {
...
+
+; tag::transfer[]
+; Transfer ownership of a key.
+kvstore.transfer@param = {
+    ; Key.
+    0 => bstr,
+
+    ; Alternative key owner
+    ; The sender must be related to this address, e.g., account, subresource, etc.
+    ? 1 => non-anonymous-address,
```
Comment (fmorency) [inline attributes/network/cddl/7_kvstore_commands.cddl:77]: Why is this needed?

**Review by fmorency — CHANGES_REQUESTED** [review]
Thanks. Minor change and a question.

Also, please add a new `canKvStoreTransfer` role in `9_account_features/cddl/2_account_kvstore.cddl`

**Review by fmorency — APPROVED** [review]

**Review by fmorency — CHANGES_REQUESTED** [review]
Attributes 11 and 12 are currently used by tokens create/mint/burn in #73

**Comment by hansl** [comment]
Thanks. I'll pick the next one (13). We should come up with a good way to pick attribute numbers.

File: attributes/network/9_account_features/cddl/2_account_kvstore.cddl [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ; tag::types[]
 account-role /= "canKvStorePut"
              /  "canKvStoreDisable"
+             /  "canKvStoreTransfer"
```
Comment (fmorency) [inline attributes/network/9_account_features/cddl/2_account_kvstore.cddl:4]: nit: We should probably create a new account feature for this role

**Review by fmorency — APPROVED** [review]
LGTM, assuming the comment has been addressed.

File: attributes/network/9_account_features/cddl/2_account_kvstore.cddl (hunk 3 above)
Comment (hansl) [reply attributes/network/9_account_features/cddl/2_account_kvstore.cddl:4]: Actually, I'm not sure. The KvStore account feature adds support for kvstore in general, to an account. Roles are specified separately? Not sure here. Like, when I create an account I have to add `kvstore` AND `kvstore_transfer` to be able to transfer keys?
